ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/ExceptionClassifier.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.helpers;
+
+import java.io.IOException;
+
+public class ExceptionClassifier {
```

Comment:
This appears to be a utility class that is never instantiated. If that's the case, then the class can be made final, and a private constructor should be added.